Sources/Core/CharcoalViewController.swift

```diff
@@ -67,6 +70,16 @@ public class CharcoalViewController: UINavigationController {
         delegate = self
     }
 
+    // MARK: - Public
+
+    public func set(selection: Set<URLQueryItem>) {
+        selectionStore = FilterSelectionStore(queryItems: selection)
```

Comment:
If you create a new instance of selection store here it needs to be injected to `rootFilterViewController` and potentially more view controllers. Even though I don't really like it, I think it would be easier to create selection store only once and them set query items by making `var queryItems` internal.

Reply:
Je, that's true. We could have a `set(selection: ...)` in FilterSelectionStore as well and keep `queryItems` private.

Reply:
Yep, that's even better 👍

Reply:
Fixed now ;)